### PR TITLE
A few miscellaneous fixes.

### LINKED ISSUE #2385: Schema swap workflow incompatible with semi-sync
I've run a few tests in our stage environment, and for the most part things work great.

However, I noticed that when the schema swap workflow is propagating backups it takes down the replica node without promoting another node to replica.

When semi-sync is enabled this causes a write lock on the master because rdonly nodes do NOT ack the master for semi-sync.

I think we should promote a rdonly node to a replica while we are updating the old replica.

### PR DESCRIPTION
**Pull request by pivanof**
1. Fix PlannedReparentShard to not panic when the shard doesn't have the master
   (MasterAlias is nil).
2. Fix documentation for schema swap to reflect the new UI and to also show how
   to start the schema swap from command line.
3. Expand documentation about Vitess replication to make it clear that with
   semi-sync each shard needs to have at least 2 replica tablets to make the
   master available in case of an outage of one of the replica tablets.
   This wasn't obvious to people setting up their production configurations.

Fixes #2385.

BUG=32724983

### CODE REVIEW

File: go/vt/wrangler/reparent.go [hunk 1]
```diff
@@ -363,7 +363,11 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 	if topoproto.TabletAliasEqual(shardInfo.MasterAlias, masterElectTabletAlias) {
 		return fmt.Errorf("master-elect tablet %v is already the master", topoproto.TabletAliasString(masterElectTabletAlias))
 	}
-	oldMasterTabletInfo, ok := tabletMap[*shardInfo.MasterAlias]
+	var oldMasterTabletInfo *topo.TabletInfo
+	ok = false
```
Comment (alainjobart) [inline go/vt/wrangler/reparent.go]: Should ok start with true? If shardInfo has no MasterAlias, then we can keep going?

Comment (pivanof) [inline go/vt/wrangler/reparent.go]: Well, you've said in the past yourself that PlannedReparentShard should work only when old master is available. :)
If we want to allow it to work without old master available we need to do more changes below. But probably when MasterAlias is nil we shouldn't assume that there's actually no writable MySQL, it's probably just something wrong with the topology (as I believe it was when the problem was reported on slack).

Comment (alainjobart) [inline go/vt/wrangler/reparent.go]: Ah makes sense, thanks for the reminder.

Then the error message below is gonna panic, so maybe special-case the nil case and return a specific error would be best.

Comment (pivanof) [inline go/vt/wrangler/reparent.go]: Do you mean fmt.Errorf("old master tablet %v is not in the shard", topoproto.TabletAliasString(shardInfo.MasterAlias)) will panic? TabletAliasString() has special handling of nil, so it shouldn't panic. Or are you talking about something else?

Comment (alainjobart) [inline go/vt/wrangler/reparent.go]: I was talking about that, yes. I forgot the special handling.

But in any case, there is just way too much thinking going into reading this code, that depend on non-obvious handling of pointers. How about:

if topoproto.TabletAliasIsZero(shardInfo.MasterAlias) {
    return fmt.Errorf("shard %v/%v has no current master" ...
)

then you don't need the rest of the change. Will be way easier to read and understand.

Comment (pivanof) [inline go/vt/wrangler/reparent.go]: Done.

**Comment by alainjobart** [comment]
LGTM

**Pull request merged**